### DISCUSSION TITLE
Engine Node Incompatible

### DISCUSSION THREAD

**Discussion by tutex24** [tinchoabbate/damn-vulnerable-defi#52] [Help / Q&A / Troubleshooting] [answered]
Hi i just clone the damnvulnerable defi repo to learn. When i try to yarn install this happens(refer picture).  What should i do?
![defi](https://github.com/tinchoabbate/damn-vulnerable-defi/assets/101985889/b74b7a10-fdcf-4721-a9a4-e08ad46691f7)

**Comment by tinchoabbate** [comment, marked as answer]
Hi! I'd recommend you switch to v4 now, which doesn't use node and instead uses Foundry for testing. The setup should be easier. Thanks!
